src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
+import "./index.module.css";
```

Comment:
Don't make index.css as index.module.css . Cz it will scope all the CSS properties of index.module.css locally by default. So we need to import that file into each jsx file to use the properties. Instead of doing that we should make it global.

Reply:
ok noted